packages/npm-mongo/package.js

```diff
@@ -3,12 +3,12 @@
 
 Package.describe({
   summary: "Wrapper around the mongo npm package",
-  version: "3.3.0-beta190.4",
+  version: "3.4.0-beta190.4",
   documentation: null
 });
 
 Npm.depends({
-  mongodb: "3.2.7"
+  mongodb: "3.4.0"
```

Comment:
Note: I took the liberty of updating the `mongodb` npm package to the latest version (@klaussner had used 3.3.2 in the original version of this PR). Maybe that was not a good idea?

Reply:
This is a good idea 😉 I believe @klaussner was going to do the same.